packages/logic-utils/api.d.ts

```diff
@@ -83,3 +83,15 @@ export function getRootSymbolParent(node: BaseUI5Node): BaseUI5Node | undefined;
  * Return a human-readable string representation of a UI5 type
  */
 export function typeToString(type: UI5Type | undefined): string;
+
+/**
+ * Check if an xml attribute key is an xmlns key
+ * @param key
+ */
+export function isXMLNamespaceKey(key: string): boolean;
```

Comment:
Maybe this belong in XML-Tools?
We already do it there (incorrectly?)
- https://github.com/SAP/xml-tools/blob/master/packages/ast/lib/build-ast.js#L259

It is very syntax specific.
- https://www.w3.org/TR/xml-names/#NT-PrefixedAttName

Reply:
It's not incorrect really, the spec says it's an attribute name that starts with `xmlns:` and doesn't have `:` after. The lower level validation of which characters can appear in the attribute name is handled by the parser in the link you put above.
This utility might belong there, yes. I don't know where to put it though. Maybe we can discuss it tomorrow?

Reply:
o.k lets sync on this.